analyses/survival-analysis/survival-analysis_tp53_telomerase.Rmd

```diff
@@ -0,0 +1,333 @@
+                             meta_indep_cancer_groups %>% mutate(OS_status = ifelse(OS_status == "LIVING", 0, 1)), 
+                             p.adjust.method = "bonferroni")
+
+```
+
+## Multivariate models 
+
+
+Background on hazard ratios and output, from [ref](http://www.sthda.com/english/wiki/cox-proportional-hazards-model):
+
+>"Hazard ratio above 1 indicates a covariate that is positively associated with the event probability, and thus negatively associated with the length of survival.
+>
+>In summary,
+>
+>HR = 1: No effect
+>HR < 1: Reduction in the hazard
+>HR > 1: Increase in Hazard" 
+>
+>The Cox regression results can be interpreted as follow:
+_Statistical significance._ The column marked "z" gives the Wald statistic value. It corresponds to the ratio of each regression coefficient to its standard error (z = coef/se(coef)). The wald statistic evaluates, whether the beta (β) coefficient of a given variable is statistically significantly different from 0. From the output above, we can conclude that the variable sex have highly statistically significant coefficients.
+>
+>_The regression coefficients._ The second feature to note in the Cox model results is the the sign of the regression coefficients (coef). A positive sign means that the hazard (risk of death) is higher, and thus the prognosis worse, for subjects with higher values of that variable. The variable sex is encoded as a numeric vector. 1: male, 2: female. The R summary for the Cox model gives the hazard ratio (HR) for the second group relative to the first group, that is, female versus male. The beta coefficient for sex = -0.53 indicates that females have lower risk of death (lower survival rates) than males, in these data.
+>
+>_Hazard ratios._ The exponentiated coefficients (exp(coef) = exp(-0.53) = 0.59), also known as hazard ratios, give the effect size of covariates. For example, being female (sex=2) reduces the hazard by a factor of 0.59, or 41%. Being female is associated with good prognostic.
+>
+>_Confidence intervals of the hazard ratios._ The summary output also gives upper and lower 95% confidence intervals for the hazard ratio (exp(coef)), lower 95% bound = 0.4237, upper 95% bound = 0.816.
+>
+>_Global statistical significance of the model._ Finally, the output gives p-values for three alternative tests for overall significance of the model: The likelihood-ratio test, Wald test, and score logrank statistics. These three methods are asymptotically equivalent. For large enough N, they will give similar results. For small N, they may differ somewhat. The Likelihood ratio test has better behavior for small sample sizes, so it is generally preferred.
+>
+> Below, HR = exp(coeff) 
+
+
+### Interaction model: `tp53_score*telomerase_score*hgg_group` 
+
+Interaction terms are generally not signifcant, except `tp53_score:telomerase_score` which is only marginally significant at `P=0.0358`. This result suggests an additive model is more appropriate.
```

Comment:
If it's easy enough to extract just the p-value from what `fit_save_model()`, you could put this after the code chunk and use  p = `r <variable with the p-value in it>`. Just a bit of future-proofing for thought.

Reply:
Done.